pkg/catalog/loader/loader.go

```diff
@@ -405,41 +403,53 @@ func (store *Store) LoadTemplatesWithTags(templatesList, tags []string) []*templ
 					stats.Increment(templates.SkippedUnsignedStats)
 					continue
 				}
-				if len(parsed.RequestsHeadless) > 0 && !store.config.ExecutorOptions.Options.Headless {
+				// if template has request signature like aws then only signed and verified templates are allowed
+				if parsed.UsesRequestSignature() && !parsed.Verified {
+					stats.Increment(templates.SkippedRequestSignatureStats)
+					continue
+				}
+				// DAST only templates
+				if store.config.ExecutorOptions.Options.DAST {
+					// check if the template is a DAST template
+					if parsed.IsFuzzing() {
+						loadedTemplates = append(loadedTemplates, parsed)
+					} else {
+						_ = parsed
```

Comment:
I guess this can be removed? (there is another one at https://github.com/projectdiscovery/nuclei/blob/0d6cff2165569e6151bd05a3a0b65acd77c77e9f/pkg/protocols/http/raw/raw.go#L286